packages/pigeon/platform_tests/shared_test_plugin_code/lib/test_types.dart

```diff
@@ -416,24 +567,23 @@ final AllTypes genericAllTypes = AllTypes(
   anEnum: AnEnum.fortyTwo,
   anObject: 1,
   list: list,
-  stringList: stringList,
-  intList: intList,
-  doubleList: doubleList,
-  boolList: boolList,
-  // aClass: genericAllNullableTypesWithoutRecursion,
-  enumList: enumList,
-  // objectList: list,
-  listList: listList,
-  // mapList: mapList,
-  map: map,
-  stringMap: stringMap,
-  intMap: intMap,
-  // doubleMap: doubleMap,
-  // boolMap: boolMap,
-  enumMap: enumMap,
-  // objectMap: map,
-  // listMap: listMap,
-  // mapMap: mapMap,
+  stringList: nonNullStringList,
+  intList: nonNullIntList,
+  doubleList: nonNullDoubleList,
+  boolList: nonNullBoolList,
+  enumList: nonNullEnumList,
+  objectList: nonNullList,
+  listList: nonNullListList,
+  // mapList: nonNullMapList,
+  map: nonNullMap,
+  stringMap: nonNullStringMap,
+  intMap: nonNullIntMap,
+  // doubleMap: nonNullDoubleMap,
+  // boolMap: nonNullBoolMap,
+  enumMap: nonNullEnumMap,
+  objectMap: nonNullMap,
+  // listMap: nonNullListMap,
+  // mapMap: nonNullMapMap,
```

Comment:
I either missed this in the review where they were added, or have forgotten; why are there commented out field here and below, vs them just not being here?

(https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abandonware)

Reply:
Just things I missed when cleaning up the pr aka future work plans